### PR TITLE
WFLY-14400 Ensure wildfly-clustering-ejb-* marshallers use valid protobuf encoding

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.redhat.com/browse/WFLY-14400

Requires: https://github.com/wildfly/wildfly/pull/13951

### CODE REVIEW

**Comment by pferraro** [comment]
@rhusar Please review

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            }
+        }
+        return new ClientMapping(sourceAddress, sourceMask, destinationAddress, destinationPort);
+    }
+
+    @Override
+    public void writeTo(ProtoStreamWriter writer, ClientMapping mapping) throws IOException {
+        InetAddress address = mapping.getSourceNetworkAddress();
+        if (!address.equals(DEFAULT_SOURCE_ADDRESS)) {
+            System.out.println(address + " != " + DEFAULT_SOURCE_ADDRESS);
```
Comment (rhusar) [inline clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:83]: Please rid of this System.out-s.

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            writer.writeObject(SOURCE_ADDRESS_INDEX, address);
+        }
+        int sourceNetworkMask = mapping.getSourceNetworkMaskBits();
+        if (sourceNetworkMask != DEFAULT_SOURCE_MASK) {
+            System.out.println(sourceNetworkMask + " != " + DEFAULT_SOURCE_MASK);
```
Comment (rhusar) [inline clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:88]: Please rid of this System.out-s.

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            writer.writeUInt32(SOURCE_MASK_INDEX, sourceNetworkMask);
+        }
+        String destinationAddress = mapping.getDestinationAddress();
+        if (!destinationAddress.equals(DEFAULT_DESTINATION_ADDRESS)) {
+            System.out.println(destinationAddress + " != " + DEFAULT_DESTINATION_ADDRESS);
```
Comment (rhusar) [inline clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:93]: Please rid of this System.out-s.

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            writer.writeString(DESTINATION_ADDRESS_INDEX, mapping.getDestinationAddress());
+        }
+        int destinationPort = mapping.getDestinationPort();
+        if (destinationPort != DEFAULT_DESINATION_PORT) {
+            System.out.println(destinationPort + " != " + DEFAULT_DESINATION_PORT);
```
Comment (rhusar) [inline clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:98]: Please rid of this System.out-s.

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java [hunk 5]
```diff
@@ -0,0 +1,107 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2013, Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:3]: This class is new enough to deserve a this year here.

**Review by rhusar — CHANGES_REQUESTED** [review]

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java (hunk 5 above)
Comment (pferraro) [reply clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:3]: This is a rename of ClientMappingExternalizer, which was written in 2013.

File: clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java (hunk 1 above)
Comment (pferraro) [reply clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/ClientMappingMarshaller.java:83]: oops. :)

**Review by rhusar — APPROVED** [review]
👍

**Pull request merged**